src/common/storage/src/operator.rs

```diff
@@ -338,25 +338,25 @@ fn init_moka_operator(v: &StorageMokaConfig) -> Result<impl Builder> {
     Ok(builder)
 }
 
-/// init_redis_operator will init a reids operator.
-fn init_redis_operator(v: &StorageRedisConfig) -> Result<impl Builder> {
-    let mut builder = services::Redis::default();
-
-    builder.endpoint(&v.endpoint_url);
-    builder.root(&v.root);
-    builder.db(v.db);
-    if let Some(v) = v.default_ttl {
-        builder.default_ttl(Duration::from_secs(v as u64));
-    }
-    if let Some(v) = &v.username {
-        builder.username(v);
-    }
-    if let Some(v) = &v.password {
-        builder.password(v);
-    }
-
-    Ok(builder)
-}
+/// init_redis_operator will init a redis operator.
```

Comment:
No need any more?

Reply:
It's better to remove redis here, Databend has no plan to use it.

Reply:
Fixed.